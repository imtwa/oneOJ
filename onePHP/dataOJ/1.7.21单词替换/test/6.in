Line 2 is the word a to be 
2
second