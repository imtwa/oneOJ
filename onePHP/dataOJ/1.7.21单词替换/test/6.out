Line second is the word a to be 